projects/lib/src/uciengine.cpp

```diff
@@ -826,9 +826,14 @@ QString UciEngine::sanPv(const QVarLengthArray<QStringRef>& tokens)
 		auto move = board->moveFromString(token.toString());
 		if (move.isNull())
 		{
-			qWarning("Illegal PV move %s from %s",
-				 qUtf8Printable(token.toString()),
-				 qUtf8Printable(name()));
+			QString tokenString(token.toString());
+			qWarning("Illegal PV move %s from %s (%s)",
```

Comment:
I think you can use `%d` instead of `%s` and `id()` instead of `qUtf8Printable(QString::number(id()))`

Reply:
Very true. I did not think about formats other than %s. Changed accordingly.